### ISSUE TITLE
General settings and Overview show blank page

### ISSUE THREAD

**Issue by ghost** [Skouat/ext_paypal_donation#44]
Clicking on General Settings and Overview in the ACP, I get nothing but blank white pages. 

Installed on 3.2

**Comment by Skouat** [comment]
Which version did you install?

**Comment by ghost** [comment]
This was the name of the zip...
Skouat-ext_paypal_donation-637f8f6
Downloaded today. 
 

    On Tuesday, April 4, 2017 4:42 PM, Mario Skouat <notifications@github.com> wrote:
 

 Which version did you install?—

**Comment by Skouat** [comment]
Uninstall the extension and use the 3.2.x version.
Use this link : https://codeload.github.com/Skouat/ext_paypal_donation/zip/develop-3.2.x

**Comment by ghost** [comment]
That worked. Thank you. 
 

    On Wednesday, April 5, 2017 8:09 AM, Mario Skouat <notifications@github.com> wrote:
 

 Uninstall the extension and use the 3.2.x version.
Use this link : https://codeload.github.com/Skouat/ext_paypal_donation/zip/develop-3.2.x—

**Comment by gtman55** [comment]
Another major bug I noticed. To test the system I made a $1 donation. The donation went through..
But...
On my page it still says there where no donations. That's a problem.  

    On Wednesday, April 5, 2017 8:23 AM, Mitch <mitchiepooh4u@yahoo.com> wrote:
 

 That worked. Thank you. 
 

    On Wednesday, April 5, 2017 8:09 AM, Mario Skouat <notifications@github.com> wrote:
 

 Uninstall the extension and use the 3.2.x version.
Use this link : https://codeload.github.com/Skouat/ext_paypal_donation/zip/develop-3.2.x—
